src/ALCProxy.Communication/Class1.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Text;
+
+namespace ALCProxy.Communication
+{
+    public class ExampleDispatch<I> : DispatchProxy
```

Comment:
Why `<I>` for generic? We generally use `<T>` or a more descriptive word, like `TValue` if relevant.